webapp/components/integrations/components/add_command.jsx

```diff
@@ -4,29 +4,40 @@
 import React from 'react';
 
 import * as AsyncClient from 'utils/async_client.jsx';
+import IntegrationStore from 'stores/integration_store.jsx';
+import TeamStore from 'stores/team_store.jsx';
 import * as Utils from 'utils/utils.jsx';
 
+import {loadTeamCommands} from 'actions/integration_actions.jsx';
 import BackstageHeader from 'components/backstage/components/backstage_header.jsx';
 import {FormattedMessage} from 'react-intl';
 import FormError from 'components/form_error.jsx';
 import {browserHistory, Link} from 'react-router/es6';
 import SpinnerButton from 'components/spinner_button.jsx';
 import Constants from 'utils/constants.jsx';
+import UpdateCommandModal from '../../update_command_modal.jsx';
```

Comment:
Can you switch this to `import UpdateCommandModal from 'components/update_command_modal.jsx';`. Just safer in case this file moves around